internal/cli/atlas/integrations/describe.go

```diff
@@ -44,25 +44,25 @@ func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
 }
 
 var describeTemplateSlack = `TYPE	API TOKEN	TEAM	CHANNEL
-{{.Type}}	{{.APIToken}}	{{.TeamName}}	{{.ChannelName}}
+{{.Slack.Type}}	{{.Slack.ApiToken}}	{{.Slack.TeamName}}	{{.Slack.ChannelName}}
 `
 var describeTemplateDatadogOpsGenie = `TYPE	API KEY	REGION
-{{.Type}}	{{.APIKey}}	{{.Region}}
+{{.GetActualInstance.Type}}	{{.GetActualInstance.ApiKey}}	{{.GetActualInstance.Region}}
```

Comment:
does this work?, GetActualInstance returns an interface so I'm not sure if you can directly access .Type without a type assertion

Reply:
Yes....Verified this manually. This works fine.